tests/spread/core24/python-hello/task.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Build and run Python-based snaps in core24
+
+environment:
+  CONFINEMENT/strict: strict
+  CONFINEMENT/classic: classic
+
+restore: |
+  cd ./"${CONFINEMENT}"
+  snapcraft clean
+  rm -f ./*.snap
+  snap remove python-hello-"${CONFINEMENT}"
+
+execute: |
+  cp -r src ./"${CONFINEMENT}"/
+  cd ./"${CONFINEMENT}"
+
+  snapcraft
+  snap install python-hello-"${CONFINEMENT}"_1.0_*.snap --dangerous --classic
```

Comment:
--classic should really be on the case of $CONFINEMENT

Reply:
updated this